src/rocm_docs/data/theme_announcement.yaml

```diff
@@ -0,0 +1,27 @@
+unreleased-branch: [
+  "develop",
+  "master",
+  "amd-master",
+  "amd-staging"
+]
+unreleased-str: "This is an unreleased version of ROCm documentation."
+old-branch: [
+  "docs-5.5.0",
+  "docs-5.4.3",
+  "docs-5.4.2",
+  "docs-5.4.1",
+  "docs-5.4.0",
+  "docs-5.3.3",
+  "docs-5.3.2",
+  "docs-5.3.0",
+  "docs-5.2.3",
+  "docs-5.2.1",
+  "docs-5.2.0",
+  "docs-5.1.3",
+  "docs-5.1.1",
+  "docs-5.1.0",
+  "docs-5.0.2",
+  "docs-5.0.1",
+  "docs-5.0.0",
+]
```

Comment:
Instead of maintaining a list like this, we can just say "You are viewing version such-and-such, click here to view the latest" on all release specific versions, i.e. any branch starting with `docs-`.

Reply:
[4f409e0](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/261/commits/4f409e071eebc8d15d15eb8dedbee7aaf600400b) checks for this

to do: link to latest version

Reply:
[03969ac](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/261/commits/03969acc246650cb60fe29f4a97c2c9f6b940163) include latest version url in announcement if not on latest